### PR TITLE
check windows

### PR DESCRIPTION
**Pull request by kevincobain2000**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -4,15 +4,17 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.17, 1.18, 1.18@latest, 1.19beta1, 1.19@dev-latest, latest, dev-latest]
-        os: [ubuntu-latest, macos-latest]
+        go-version: [1.17]
```
Comment (juev) [inline .github/workflows/build.yml:7]: Why 1.17?

**Comment by juev** [comment]
As it seems to me, these lines:
https://github.com/kevincobain2000/action-gobrew/blob/master/action.yml#L17-L19

do not worked on windows. What do you think?

File: .github/workflows/build.yml (hunk 1 above)
Comment (kevincobain2000) [reply .github/workflows/build.yml:7]: Just testing. So reduced matrix as I expected to fail anyways

**Comment by kevincobain2000** [comment]
Yea, I have tbh no idea about windows.

**Comment by beppler** [comment]
Interesting, I have the same change you made on a branch here and it worked:

**Comment by kevincobain2000** [comment]
probably failing at `gobrew version` on windows.

**Comment by kevincobain2000** [comment]
Oddly enough, gobrew version consumes lot of time on Mac

```
╰─$ time gobrew version
[INFO] gobrew version is 1.7.9
gobrew version  0.01s user 0.01s system 0% cpu 6.308 total
```

**Comment by kevincobain2000** [comment]
ah because `gobrew version` calls github api to get the latest version and can also lead to api rate limit exceeded, making it fail

**Comment by kevincobain2000** [comment]
Raised issue here #94

**Comment by kevincobain2000** [comment]
This pull req is ready for review and merge

**Comment by kevincobain2000** [comment]
> Interesting, I have the same change you made on a branch here and it worked:
https://github.com/beppler/gobrew/actions/runs/3939062367

Because my workflow had `gobrew version` which failed due to rate limit exceeded as it calls `api.github..`, therefore your workflow passed with no worries.

**Comment by juev** [comment]
Very intresting

**Review by juev — APPROVED** [review]

**Pull request merged**